### ISSUE TITLE
Pandas still seems required with version 1.0.0

### ISSUE THREAD

**Issue by azmeuk** [migraf/fhir-kindling#195]
Hi. After having updated to fhir-kindling 1.0.0, on new environments my unit tests fail with an `ImportError` about pandas. Manually installing pandas solves the issue:

```
E           _pytest.nodes.Collector.CollectError: ImportError while importing test module '/woodpecker/src/forge.yaal.coop/lum1/lum1/tasks/tests/test_update_finess.py'.
E           Hint: make sure your test modules/packages have valid Python names.
E           Traceback:
E           /usr/local/lib/python3.11/importlib/__init__.py:126: in import_module
E               return _bootstrap._gcd_import(name[level:], package, level)
E           tasks/tests/test_update_finess.py:8: in <module>
E               from tasks.finess import filter_entities
E           tasks/finess.py:8: in <module>
E               from fhir_kindling import FhirServer
E           /root/.cache/pypoetry/virtualenvs/lum1-SNzkjssz-py3.11/lib/python3.11/site-packages/fhir_kindling/__init__.py:3: in <module>
E               from .fhir_server import FhirServer
E           /root/.cache/pypoetry/virtualenvs/lum1-SNzkjssz-py3.11/lib/python3.11/site-packages/fhir_kindling/fhir_server/__init__.py:1: in <module>
E               from .fhir_server import FhirServer
E           /root/.cache/pypoetry/virtualenvs/lum1-SNzkjssz-py3.11/lib/python3.11/site-packages/fhir_kindling/fhir_server/fhir_server.py:37: in <module>
E               from fhir_kindling.serde.json import json_dict
E           /root/.cache/pypoetry/virtualenvs/lum1-SNzkjssz-py3.11/lib/python3.11/site-packages/fhir_kindling/serde/__init__.py:1: in <module>
E               from .flatten import flatten_resource, flatten_resources, flatten_response
E           /root/.cache/pypoetry/virtualenvs/lum1-SNzkjssz-py3.11/lib/python3.11/site-packages/fhir_kindling/serde/flatten.py:4: in <module>
E               import pandas as pd
E           E   ModuleNotFoundError: No module named 'pandas'
```

The faulty line seems to be:

https://github.com/migraf/fhir-kindling/blob/373413cb36ff1b9ca40fe657a563de893d5b9846/fhir_kindling/serde/flatten.py#L4

**Comment by migraf** [comment]
just published `v1.0.1` which should fix this issue

**Comment by azmeuk** [comment]
It works. Thank you!
